tests/callbacks/test_EnqueuedEventTimer.cpp

```diff
@@ -22,44 +23,93 @@ namespace Kokkos::utils::tests::callbacks
 
 using namespace Kokkos::utils::callbacks;
 
-struct EnqueuedEventTimerTest : public ::testing::Test
+template <typename TimerType>
+struct TimerTest : public ::testing::Test
 {
-public:
-    using timer_t = EnqueuedEventTimer<execution_space>;
-
 public:
     void SetUp() override {
         this->exec = Kokkos::Experimental::partition_space(execution_space{}, 1)[0];
         this->dev_id = Kokkos::Tools::Experimental::device_id(this->exec);
-        this->timer = timer_t{this->exec};
+        this->timer = TimerType{this->exec};
     }
 
 protected:
     execution_space exec {};
     uint32_t dev_id = 0;
-    timer_t timer {};
+    TimerType timer {};
 };
 
-//! @test Check @ref Kokkos::utils::callbacks::EnqueuedEventTimer.
+struct EnqueuedEventTimerTest : public TimerTest<EnqueuedEventTimer<execution_space>> {};
+
+//! @test Check the duration reported by @ref Kokkos::utils::callbacks::EnqueuedEventTimer.
```

Comment:
```suggestion
//! @test Check the enqueued event duration reported by @ref Kokkos::utils::callbacks::EnqueuedEventTimer.
```

Reply:
Let's not do it because there's only one duration reported by this one, no need to disembiguate.